lib/rest2.js

```diff
@@ -1340,6 +1340,58 @@ class RESTv2 {
       .then(_takeResNotify)
   }
 
+  /**
+   * Submit multiple orders.
+   *
+   * @param {order[]} orders - list of orders (can be object literal or Order class instance)
+   * @param {Function} [cb]  - callback
+   * @returns {Promise} p
+   */
+  submitOrderMulti (orders = [], cb) {
+    const payload = orders.map((order) => {
```

Comment:
This idea look great! 
Would just add a first check as to prevent code breaking. 
if (!Array.isArray(orders)) {
  const e = new Error('orders should be an array')
  return this._cb(e, null, cb)
}